### PR TITLE
Bug 1145756 - Set the cached value r=eragonj

### PR DESCRIPTION
**Pull request by crh0716**
https://bugzilla.mozilla.org/show_bug.cgi?id=1145756

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b3cae151214db3db329c059694a3cf2a71568981

File: apps/settings/js/panels/homescreen/homescreen.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ define(function(require) {
           this.cols = value;
           this._isUpdating = false;
           if (this._cachedColsValue) {
-            var cachedValue = this._cachedValue;
-            this._cachedValue = null;
+            var cachedValue = this._cachedColsValue;
+            this._cachedColsValue = null;
```
Comment (EragonJ) [inline apps/settings/js/panels/homescreen/homescreen.js]: ahhhhhh

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#QQF90ApyQzq7XyyH8qJ7ag

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a49c91008ccec2b0d154abaa4f83c991039ea530

**Pull request merged**